lib/lita/handlers/github_pinger.rb

```diff
@@ -139,7 +139,7 @@ def act_on_comment(body, response)
           puts "Found @mentions in the body of the comment! Extracting usernames... "
 
           # get each @mentioned engineer in the comment
-          mentions = comment.split("@")[1..-1].map { |snip| snip.split(" ").first.gsub(/[^0-9a-z ]/i, '') }
+          mentions = comment.split("@")[1..-1].map { |snip| snip.split(" ").first.gsub(/[^0-9a-z\- ]/i, '') }
```

Comment:
@tommi-lew: The `\` escaping of the `-` isn't necessary, see this [Rubular example](http://rubular.com/r/RpHUAddEoR). 😉